modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -15,7 +15,7 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 * Verify that you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus and Alertmanager both have two replicas, you need four PVs to support the entire monitoring stack. The PVs are available from the Local Storage Operator, but not if you have enabled dynamically provisioned storage.
 
-* Use the block type of storage.
+* Use the `Filesystem` type of storage.
```

Comment:
I think change to below is better
Use the `Filesystem` volumeMode of storage

Reply:
> I think change to below is better Use the `Filesystem` volumeMode of storage since we are talking about volumeMode here,

Changed to "Use `Filesystem` as the storage type for `volumeMode` when configuring the persistent volume"